Makefile

```diff
@@ -158,6 +158,10 @@ debug: manager
 
 ##@ Time fixes
 
+.PHONY: csv-date
+csv-date: ## Set createdAt date in the CSV.
+	sed -r -i "s|createdAt: .*|createdAt: `date '+%Y-%m-%d %T'`|;"  ./config/manifests/bases/$(OPERATOR_NAME)operator.clusterserviceversion.yaml
```

Comment:
why this? what's the value of having the date in this file?

Reply:
> - Is the order of targets aligned to what operator-sdk generates for a new project?

Yes, it is aligned, and I have checked it again with operator-sdk version `1.11.0`. The only difference I see is the use of `catalog` instead of `index` for `catalog-build`, and `catalog-push`.

> - I prefer to further check and merge this after doing the release. Changes in the Makefile potentially affect build, CI and releases, up- and downstream...

I agree with waiting  (and the other PRs), it is not urgent.

> why this? what's the value of having the date in this file?

- For making a "stamp" of when was the last time we have run `make bundle` (and tried to build a bundle), even though we can look at the same annotation of CSV in the bundle.
- Making consistency between CSV under config with the one under bundle, and save suspicion - if there is time difference (of daysת and even more) it seems like a sign for the admin that he haven't been using the latest CSV (under config) for creating the other CSV (under bundle). 
As the admin, you (should) know that this suspicion is wrong, but maybe it could help other peers to not wonder about it. For me it would have helped :)

Reply:
> For making a "stamp" of when was the last time we have run make bundle

make bundle runs in CI on every PR, and for every build. No need to keep track when the developer run it IMHO. The field name is createdAt (when was the build) and not committedAt.

>  consistency between CSV under config with the one under bundle, and save suspicion

it's empty on both in git, that is consitent :) It is only filled in the bundle on build. 

>  a sign for the admin ... as the admin

I do not understand who the admin is in the dev/build/release process, and what his issues are?